concepticondata/conceptlists/Verheyen-2019-1000.tsv

```diff
@@ -0,0 +1,1001 @@
+Verheyen-2019-1000-74	74	beperkt	limited			9.45	3	4.3	3.1	5.1	2.95	3.05	7	2	63368	0	2357	270	2.16433293965715	0.000548804862426
+Verheyen-2019-1000-75	75	bergachtig	mountainous	MOUNTAINOUS	3755	7.8	4.8	5.65	4.8	2.7	6.6	3.6	10	3	96827	0	85	8	0.092028559513635	1.37066167032502E-05
+Verheyen-2019-1000-76	76	beroemd	famous			5.7	5.45	5	5.35	5.25	5.85	4.6	7	2	71959	2	1353	2007	2.31968236829124	0.000291717936312
+Verheyen-2019-1000-77	77	beschikbaar	available			8.75	4.25	4.4	3.95	5.05	3.4	4.55	11	3	74481	0	2035	723	0.152208672323134	4.53730666413621E-05
+Verheyen-2019-1000-78	78	besmeurd	black			9.52631578947369	4.52631578947368	4.75	3.75	2.9	5.2	2.5	8	2	38696	0	0	0	0.029744401808838	4.34758782420281E-06
+Verheyen-2019-1000-79	79	best	best			5.2	5.55	4.1	5.8	5.6	3.5	5.95	4	1	34613	16	2885	13066	1.04597302363008	0.000322541552982
+Verheyen-2019-1000-80	80	bestaand	living	BE ALIVE	1422	8.4	3.25	4.45	3.95	4.6	2.4	4.55	8	2	81014	0	1951	1	0.100759975374276	2.05143664998312E-05
+Verheyen-2019-1000-81	81	beter	better			4.9	4.9	3.85	5.45	5.6	3.3	6.05	5	2	75443	9	5094	0	3.31000052059426	0.000612840665722
+Verheyen-2019-1000-82	82	betrokken	concerned			8.85	4.5	3.3	4.85	4.31578947368421	2.85	3.85	9	3	76773	2	2058	181	0.454906269134945	0.000210278764988
+Verheyen-2019-1000-83	83	betrouwbaar	safe			8.5	3.65	5.1	4.7	5.85	4	6	11	3	66174	0	1104	665	1.41362416438541	0.000169552670961
+Verheyen-2019-1000-84	84	betweterig	pedantic			9.8	4.8	4.55	4.85	3.65	4.15	2.2	10	4	108198	1	23	15	0.136770810433979	5.37916367770004E-05
+Verheyen-2019-1000-85	85	bevallig	pretty			11.35	4.72222222222222	3.44444444444444	3.78947368421053	1.75	3.25	4.1	8	3	42149	0	103	29	0.144175556440218	1.27921914198662E-05
+Verheyen-2019-1000-86	86	bevreesd	apprehensive			8.7	5.05	4.4	3.6	4.05	4.15	2.55	8	2	45666	3	195	38	0.041649080278351	1.37261417982691E-05
+Verheyen-2019-1000-87	87	bewolkt	clouded			6.45	2.8	4.95	3.35	4.6	6.45	2.85	7	2	19217	2	59	64	0.307972705195491	2.43803353135685E-05
+Verheyen-2019-1000-88	88	bewust	aware			8.85	4.5	3.6	4.6	5.7	3.8	4.75	6	2	30945	1	6003	862	1.14033314845811	0.000284021794692
+Verheyen-2019-1000-89	89	bezet	busy	BUSY	2755	6.2	4.2	5.1	4.5	4.95	4.8	3.4	5	2	21122	5	563	201	0.395435431380257	0.000107095146179
+Verheyen-2019-1000-90	90	bezig	busy	BUSY	2755	4.6	5.55	4.65	4.55	5.75	4.65	4.05	5	2	19105	7	7954	8602	1.13103032398977	0.000169204473434
+Verheyen-2019-1000-91	91	bezitterig	possessive			9.3	5.3	4.25	5.4	3.8	3.8	2.05	10	4	99813	0	0	36	0.259325346397403	3.52102546840378E-05
+Verheyen-2019-1000-92	92	bezorgd	afraid			7.55	4.75	4.35	3.65	5.25	5.05	4.15	7	2	23651	0	1037	1976	0.767920078549726	0.000127489107926
+Verheyen-2019-1000-93	93	Bijbels	biblical			7.15	3	4.35	3.4	1.95	4.2	3.1	7	2	43611	0	443	151	0.08352531429712	1.42663360937913E-05
+Verheyen-2019-1000-94	94	bijdehand	bright	BRIGHT	679	11.75	4.5	3.6	4.78947368421053	3.25	2.9	3.15	9	3	59264	0	26	236	0.198328174790019	4.32350687367953E-05
```

Comment:
bijdehand means bright only in the “clever” sense

Reply:
CLEVER 1310 ?

Reply:
The translation into German is "klug", right? Therefore, I would map it to SMART 2530.

Reply:
I think both concepts are not wrong for this word, but judging from the description, CLEVER 1310 seems to be a better fit. To me - and again, I'm not a native speaker, so take it with a grain of salt - bijdehand has this undertone of being sly.

Reply:
I'll leave it unmapped. I think this would need a more fine-grained concept something with the meaning of "helle, gescheit, aufgeweckt".